From 2b6df60b28376e19a89aa9a5bdb1285fa0f7f30e Mon Sep 17 00:00:00 2001
From: "deepak.s" <deepak.s@adlinktech.com>
Date: Fri, 29 Mar 2024 14:43:51 +0530
Subject: [PATCH] Description of the changes

---
 backgrounds/Makefile.am | 9 +--------
 1 file changed, 1 insertion(+), 8 deletions(-)

diff --git a/backgrounds/Makefile.am b/backgrounds/Makefile.am
index 04342c7..6f6e802 100644
--- a/backgrounds/Makefile.am
+++ b/backgrounds/Makefile.am
@@ -4,13 +4,6 @@
 backgroundsdir = $(datadir)/backgrounds/xfce
 
 backgrounds_DATA = \
-    xfce-blue.jpg \
-    xfce-stripes.png \
-    xfce-verticals.png \
-    xfce-teal.jpg \
-    xfce-shapes.svg \
-    xfce-flower.svg \
-    xfce-leaves.svg
-
+	adlink.jpg
 
 EXTRA_DIST = $(backgrounds_DATA)
-- 
2.43.2

